components/TokenActions/index.tsx

```diff
@@ -33,11 +38,33 @@ export const TokenActions = ({
         logo: LogoTicker;
         text: string;
     }[];
+    isImported?: boolean;
 }): JSX.Element => {
+    const removePool = useStore(selectRemovePool);
     const { network } = useStore(selectWeb3Info, shallow);
     const provider = useStore(selectProvider);
     const [isOpen, setIsOpen] = useState<boolean>(false);
     const buttonStyles = 'flex cursor-pointer items-center p-2 text-sm hover:bg-theme-button-bg-hover w-full';
+
+    const removeImportedPoolFromUi = useCallback((): void => {
+        if (poolAddress) {
+            // Check for imported Pool in localStorage
+            const localStoragePoolAddresses = localStorage.getItem('importedPools');
+            const parsedImportedPools = localStoragePoolAddresses && JSON.parse(localStoragePoolAddresses);
+            const poolIndex = parsedImportedPools?.findIndex((pool: string) => pool === poolAddress);
```

Comment:
nitpicky but this function exists: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
I'm aware of includes() but we would still have to find the index of the pool in the localStorage array to remove it anyway, that's why I used findIndex. Also acts as a check to see if it exists